src/AuditableTraitObserver.php

```diff
@@ -43,4 +43,16 @@ public function updating(Model $model)
             $model->updated_by = $this->getAuthenticatedUserId();
         }
     }
+
+    /**
+     * Model's deleting event hook
+     *
+     * @param Model $model
+     */
+    public function deleting(Model $model)
+    {
+        if (! $model->deleted_by) {
```

Comment:
We also need some check here to check if the model has `deleted_by` column defined. Else, this will surely give us an error were `deleted_by` column does not exists?